src/Resources/views/Form/Type/sonata_type_model_autocomplete.html.twig

```diff
@@ -148,7 +148,11 @@ file that was distributed with this source code.
             // NEXT_MAJOR: Remove this BC layer while upgrading to v4.
             var templateResult = function (item) {
                 return {% block sonata_type_model_autocomplete_dropdown_item_format -%}
-                    jQuery('<div class="{{ dropdown_item_css_class }}">').text(item.label).prop('outerHTML')
+                    {% if safe_label is defined and safe_label %}
```

Comment:
so how about `safe_label|default(false)`?

Reply:
That's a new way of writing it at first it does not look like the same outcome, but there are similar if function statement written as
`{%- if read_only is defined and read_only %}`
on line 15, 21, 77

I can amend but would it not be consistent?

Reply:
I think it's ok to amend, if we choose to be consistent, we will never improve anything. If you want to go the extra mile, change the other usages too.